src/lib/components/Feedback/index.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * @fileoverview Element that handles feedback on posts.
+ */
+import {html} from 'lit-element';
+import {BaseElement} from '../BaseElement';
+import {trackEvent} from '../../analytics';
+import './_styles.scss';
+
+/**
+ * Catches feedback submission on posts to submit to Analytics.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class Feedback extends BaseElement {
+  constructor() {
+    super();
+    this.submitted = false;
+    this.submit = this.submit.bind(this);
+  }
+
+  submit(event) {
+    event.preventDefault();
+    const form = new FormData(event.target);
+    const formIterable = Array.from(form.entries());
+    if (!formIterable.length || this.submitted) {
+      return;
+    }
+
+    this.submitted = true;
+    this.requestUpdate();
+
+    formIterable.forEach((entry) => {
+      trackEvent({
+        category: 'Feedback',
+        action: 'submit',
+        label: entry[0],
+        value: Number(entry[1]),
+      });
+    });
+  }
+
+  render() {
+    // Because we share CSS with the `app.css` we want this to be in the light DOM
+    return html`
+      <details class="w-details">
+        <summary class="w-details__summary">
+          <h2 class="w-details__header">Give feedback</h2>
+        </summary>
+        <div
+          class="w-display--flex w-justify-content--center ${!this.submitted &&
+            'hidden'}"
+        >
+          <div class="w-subscribe--padded">
+            <p class="w-text--center w-mt--non">
+              Thank you for the feedback!
+            </p>
+          </div>
+        </div>
+        <form
+          class="w-feedback__form ${this.submitted && 'hidden'}"
+          @submit=${this.submit}
+        >
+          <small>All fields optional</small>
+          <div class="web-feedback__rows">
+            <div class="web-feedback__row">
+              <div></div>
+              <div>Yes</div>
```

Comment:
```suggestion
              <div id="yes-label">Yes</div>
```

Need this for the `aria-labelledby` attributes to work.